recipe/ssl.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Deployer;
+
+desc('Create the SSL certificate for the staging url');
+task(
+    'sumo:ssl:create',
+    function () {
+        $command = sprintf(
+            'create_ssl %1$s.%2$s.%3$s',
+            get('project'),
+            get('client'),
+            'php' . str_replace('.', '', get('php_version'))
```

Comment:
Klopt het dat de php versie hier de laatste parameter is? Moet niet prefix zijn?

Reply:
command `create_ssl` verwacht iets als app.getuigenonderweg.php81 als parameter.
en `get('php_version')` returned `8.1`
Of begrijp ik je verkeerd?

Reply:
nope, ideaal